### DISCUSSION TITLE
About applying for the github student pack for middle school students

### DISCUSSION THREAD

**Discussion by Student-Jasons** [community/community#136276] [GitHub Education]
### Hi everyone,

hello. I'm an enabler living in South Korea.
I am currently a middle school student and would like to ask if you have any questions about the application.

1. my school is not your official registrar and I would like to ask what documents I need to register for this, and what steps I need to take if I want to register as an official registrar for other developers who attend my school. For your information, our school issues all students an email address in the form of jp.s******@ggm.goe.go.kr and the student ID card is in Korean.

2. I would like to ask if it is possible to apply for middle school students who are over 13 years old.

3. I am currently 12 years old, but my birthday is August 26, so I will be 13 years old in a week from the date of this post. I would like to ask if you can issue the documents in advance, apply for it, and then approve it when I turn 13.

Translated with DeepL.com (free version)

**Comment by Student-Jasons** [comment]
For reference, if you want to comment more on this, jp.s******@ggm.goe.go.kr wholesalers are 
jp(abbreviation of Jeongpyeong Middle School, the name of the school).s(student)******(unique ID of students and staff)@ggm(abbreviation of middle school).goe.go.kr(government wholesaler)

**Reply by 594ppx** [reply]
well,I'm a 13-year-old student too!Thanks for your question.
But......my DNS is so slow😭
Can github improve this,please？

**Comment by BONY-SL** [comment]
GitHub requires that students be at least 13 years old to apply for the GitHub Student Developer Pack. Since you will be turning 13 soon, you should wait until you are officially 13 before applying

If your school is not listed as an official registrar on GitHub, you can still apply as an individual. However, if you want to register your school for other students to benefit as well, you would need to contact GitHub support for guidance on the process. They may require additional information or documentation from your school administration.

**Reply by Student-Jasons** [reply]
Thank you!
